tests/tests.py

```diff
@@ -801,7 +801,8 @@ def test_open_save(self):
             os.path.join(base_dir, 'test4_hex.kdbx'),       # legacy 64 byte hexadecimal keyfile test
             os.path.join(base_dir, 'test3.kdbx'),           # KDBX v3 transformed_key open test
             os.path.join(base_dir, 'test4_hex.kdbx'),       # KDBX v4 transformed_key open test
-            stream
+            stream,
+            os.path.join(base_dir, 'extra_content.kdbx')    # has trailing bytes after the xml
```

Comment:
Maybe a more accurate name is `test4_uncompressed.kdbx`?

Reply:
In my test, not all uncompressed database have extra content.